examples/account_management/reject_merchant_center_link.rb

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Demonstrates how to reject a Merchant Center link request.
+#
+# Prerequisite: You need to have access to a Merchant Center account. You can
+# find instructions to create a Merchant Center account here:
+# https://support.google.com/merchants/answer/188924.
+#
+# To run this example, you must use the Merchant Center UI or the Content API
+# for Shopping to send a link request between your Merchant Center and
+# Google Ads accounts.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def reject_merchant_center_links(customer_id, merchant_center_account_id)
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  merchant_center_link_service = client.service.merchant_center_link
+
+  # Rejects a pending link request or unlinks an enabled link for a Google Ads
+  # account with customer_id from a Merchant Center account with
+  # merchant_center_account_id.
+  response = merchant_center_link_service.list_merchant_center_links(
+    customer_id: customer_id,
+  )
+  puts "#{response.merchant_center_links.size} Merchant Center link(s) found " \
+    "with the following details:"
+
+  response.merchant_center_links.each do |link|
```

Comment:
You could probably use response.merchant_center_links.detect {|link| link.id == merchange_center_account_id.to_i} to extract the one value you're actually interested in, and then act on that one, rather than needing to use a break below.

Reply:
Changed to using detect. Thanks for the suggestion!